tests/flit_makefile/tst_incremental_build.py

```diff
@@ -139,7 +139,7 @@
 ['devrun', 'obj/dev/ALL-FLIT.cpp.o', 'obj/dev/Empty.cpp.o', 'obj/dev/main.cpp.o']
 
 >>> touched_files(after_build)
```

Comment:
Please insert a comment before this why this is needed and the issue number for fixing it.

Reply:
Done.